roles/readarr/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Create Readarr Directories
+  file:
+    path: "{{ item }}"
+    state: directory
+  with_items:
+    - "{{ readarr_data_directory }}"
+    - "{{ readarr_data_directory }}/config"
+
+- name: Create Readarr Docker Container
+  docker_container:
+    name: readarr
+    image: linuxserver/readarr:nightly
+    pull: true
+    volumes:
+      - "{{ readarr_data_directory }}/config:/config:rw"
+      - "{{ books_root }}:/books:rw"
```

Comment:
You need to use the new variable you created - I wouldn't expect to see `books_root` here.

Reply:
done